src/infoview.ts

```diff
@@ -36,26 +38,34 @@ class InfoDocument implements Disposable {
     }
 
     get text() {
-        return this.lines.join('\n');
+        return this.contents;
     }
 
     dispose() {
         for (const s of this.subscriptions) s.dispose();
     }
 
     private render() {
-        this.lines = [];
-        this.renderHeader();
-        this.renderGoal();
-        this.renderMessages();
+        this.contents = this.renderHeader() + this.renderGoal() + this.renderMessages();
+    }
+
+    private stylesheet() {
+        return `#settings { display: block; text-align: right; }
+            h1 {
+                display: block;
+                background-color: blue;
```

Comment:
Absolutely :-) Is there an easy access to the style colors? Or should we just provide a light and a dark version? (like the progress images)